src/sestest/NodeEntitlementsBuilder.cs

```diff
@@ -184,6 +197,16 @@ private IEnumerable<Errorable<IPAddress>> Addresses()
             }
         }
 
+        private static Errorable<IPAddress> TryParseIPAddress(string address)
+        {
+            if (IPAddress.TryParse(address, out var ip))
+            {
+                return Errorable.Success(ip);
+            }
+
+            return Errorable.Failure<IPAddress>($"IP address '{address}' not in expected format (IPv4 and IPv6 supported).");
```

Comment:
"[is] not in [an] expected format" for readability :)

Reply:
Done.